File: Latihan B/Python/TimSepakBola.py
class TimSepakBola:

	#atribut private
	nama_tim = "-"
	negara = "-"
	tahun_berdiri = "-"
	pemain = []
	pelatih = "-"

	#konstruktor
	def __init__(self):
		self.nama_tim = "-"
		self.negara = "-"
		self.tahun_berdiri = "-"
		self.pemain = []
		self.pelatih = "-"

	#getter dan setter nama tim
	def setNamaTim(self, nama_tim):
		self.nama_tim = nama_tim

	def getNamaTim(self):
		return self.nama_tim

	#getter dan setter asal negara tim
	def setNegara(self, negara):
		self.negara = negara

	def getNegara(self):
		return self.negara

	#getter dan setter tahun berdiri tim
	def setTahunBerdiri(self, tahun_berdiri):
		self.tahun_berdiri = tahun_berdiri

	def getTahunBerdiri(self):
		return self.tahun_berdiri

	#getter dan setter pemain
	def setPemain(self, indeks, pemain):
		self.pemain.insert(indeks, pemain)

	def getPemain(self, indeks):
		return self.pemain[indeks]

	#getter dan setter pelatih
	def setPelatih(self, pelatih):
		self.pelatih = pelatih

	def getPelatih(self):
		return self.pelatih

File: Latihan B/Java/TimSepakBola.java
public class TimSepakBola{

	//atribut private
	private String nama_tim;
	private String negara;
	private String tahun_berdiri;
	private String pemain[] = new String[50];
	private String pelatih;

	//konstruktor
	public TimSepakBola(){

	}

	//getter dan setter nama tim
	public void setNamaTim(String nama_tim){
		this.nama_tim = nama_tim;
	}

	public String getNamaTim(){
		return this.nama_tim;
	}

	//getter dan setter asal negara tim
	public void setNegara(String negara){
		this.negara = negara;
	}

	public String getNegara(){
		return this.negara;
	}

	//getter dan setter tahun berdiri tim
	public void setTahunBerdiri(String tahun_berdiri){
		this.tahun_berdiri = tahun_berdiri;
	}

	public String getTahunBerdiri(){
		return this.tahun_berdiri;
	}

	//getter dan setter pemain
	public void setPemain(int indeks, String pemain){
		this.pemain[indeks] = pemain;
	}

	public String getPemain(int indeks){
		return this.pemain[indeks];
	}

	//getter dan setter pelatih
	public void setPelatih(String pelatih){
		this.pelatih = pelatih;
	}

	public String getPelatih(){
		return this.pelatih;
	}

}